### ISSUE TITLE
Assign to same false negative.

### ISSUE THREAD

**Issue by JohanLarsson** [JohanLarsson/Gu.Analyzers#6]
```C#
return new Foo { Bar = Bar };
```
